pkg/scaffold/cmd.go

```diff
@@ -129,4 +135,18 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func setupMetrics(mgr manager.Manager) {
+	// Expose metrics by creating a Service object
+	s, err := metrics.ExposeMetricsPort(metricsAddress)
+	if err != nil {
+		log.Info(err.Error())
```

Comment:
Instead of logging here, should we return the errors and error handling/logging in the calling code?

Reply:
Yes, thanks, this was done before the move, will do 👍